cli/common/terminal/basic.go

```diff
@@ -147,3 +157,31 @@ func (ui *basicUI) NamedValues(rows []NamedValue, opts ...Option) {
 func (ui *basicUI) OutputWriters() (io.Writer, io.Writer, error) {
 	return os.Stdout, os.Stderr, nil
 }
+
+// interpret decomposes the msg and arguments into the message, style, and writer.
+func (ui *basicUI) interpret(msg string, raw ...interface{}) (string, string, io.Writer) {
```

Comment:
`parse` is usually the verbiage utilized for similar behavior. `interpret` seemed a little confusing.